docs/markdown/Reference-manual.md

```diff
@@ -673,6 +674,12 @@ be passed to [shared and static libraries](#library).
 - `pie` *(since 0.49.0)*: build a position-independent executable
 - `native`: is a boolean controlling whether the target is compiled for the
   build or host machines. Defaults to false, building for the host machine.
+- `win_subsystem` *(since 0.56.0)* specifies the subsystem type to use
+  on the Windows platform. Typical values include `console` for text
```

Comment:
Unclear if numeric values are expected to work here, so perhaps `is a string specifying the subsystem...`?